content/docs/authorization/cli.md

```diff
@@ -247,6 +247,11 @@ Usually, you will want to pipe the output to kubectl to apply the secret
 ```bash
 karavictl generate token --tenant Alice --admin-token admintoken.yaml --addr csm-authorization.host.com | kubectl apply -f -
 ```
+The token is read once when the driver pods are started and is not dynamically updated. If you are applying a new token in an existing driver installation, restart the driver pods for the new token to take effect.
+```bash
+kubectl -n <driver-namespace> rollout restart deploy/<driver>-controller
+kubectl -n <driver-namespace> rollout restart ds/<driver>-node
+```
```

Comment:
Let's also add this to the troubleshooting page

Reply:
There's already a section in the Troubleshooting page that includes that. Is there a different section I should add that to on that page?